tools/mo/openvino/tools/mo/front/tf/fifo_replacer.py

```diff
@@ -113,3 +120,52 @@ def replace_sub_graph(graph: Graph, match: dict, **kwargs):
         graph.remove_node(match['queue_deque'].id)
         graph.remove_node(match['fifo_queue'].id)
 
+
```

Comment:
please update `Supported_Frameworks_Layers.md` with `QueueDequeue` and `QueueDequeueV2` operations with a comment `Supported only when it is part of a sub-graph of the special form`.

Reply:
Done